generation/maisi/scripts/diff_model_create_training_data.py

```diff
@@ -175,10 +176,10 @@ def diff_model_create_training_data(env_config_path: str, model_config_path: str
 
     autoencoder = define_instance(args, "autoencoder_def").to(device)
     try:
-        checkpoint_autoencoder = torch.load(args.trained_autoencoder_path)
+        checkpoint_autoencoder = torch.load(args.trained_autoencoder_path, weights_only=True)
         autoencoder.load_state_dict(checkpoint_autoencoder)
     except Exception:
-        logger.error("The trained_autoencoder_path does not exist!")
+        logger.info("The trained_autoencoder_path does not exist!")
```

Comment:
Hi @dongyang0122, could you please verify if this change makes sense? It seems that even if the trained_autoencoder_path hasn't been loaded, it doesn't affect the execution. Therefore, I’m changing it from an error to an info message to avoid confusion. Thanks.

Reply:
if the trained_autoencoder_path hasn't been loaded, the result will be wrong

Reply:
Ok, then maybe let me remove this try-except to avoid confusion.

Reply:
Updated, please take a look at it again, thanks.

Reply:
Thanks. It should be `logger.error` here.

Reply:
yes, I have revert it to logger.error, thanks.